src/apps/media/src/app-revamp/components/EmptyState.tsx

```diff
@@ -0,0 +1,17 @@
+import { Typography } from "@mui/material";
+import { Stack } from "@mui/material";
+import { FC } from "react";
+import { UploadButton } from "./UploadButton";
+
+export type EmptyState = {};
```

Comment:
I'd agree that we probably do not need to export the type and I will change it to a local `Props` type.

Generally, if we DO export the type, I prefer to give it the same name as the component so that the consumer will automatically import the type when the component is imported. I feel that this is more ergonomic for the consumer, and I don't think it creates any confusion because TS types and JSX components are not interchangeable. A decent IDE will show the programmer which is which if they get confused.